eox_essence/settings/common.py

```diff
@@ -33,9 +33,32 @@
 
 USE_TZ = True
 
-
-def plugin_settings(settings):  # pylint: disable=unused-argument
+EOX_ESSENCE_ENROLLMENTS = {
+    'model': {
+        'backend': 'student.models',
+        'name': 'CourseEnrollment',
+        'get': 'get_enrollment',
+        'allowed_parameters': [
+            'course_key',
+            'user',
+        ],
+    },
+    'serialized': {
+        'backend': 'openedx.core.djangoapps.enrollments',
```

Comment:
I suggest that we bring both this API and the serializer to this plugin.
When we look at the other models we want to implement: user, course, certificate, they have no API at edx-platform so wer are going to have to write a bunch of APIs either way and we would not depend on the "openedx.core.djangoapps.enrollments.api" being stable